documentation/samples/snippets/sync_table.ts

```diff
@@ -2,44 +2,32 @@ import * as coda from "@codahq/packs-sdk";
 
 const pack = coda.newPack();
 
-// BEGIN
+const $3$MySchema$ = undefined;
 
-const MySchema = coda.makeObjectSchema({
-  properties: {
-    property1: { type: coda.ValueType.String },
-    property2: { type: coda.ValueType.Number },
-    // Add more properties here.
-  },
-  displayProperty: "property1", // Which property above to display by default.
-  idProperty: "property2", // Which property above is a unique ID.
-});
+// BEGIN
 
 pack.addSyncTable({
-  name: "<User-visible name for the sync table>",
-  description: "<User-visible description for the sync table>",
-  identityName: "<User-visible name for the column containing the schema>",
-  schema: MySchema,
+  name: "${1:MyThings}",
+  description: "${2:Table description.}",
+  identityName: "$1",
+  schema: $3$MySchema$,
   formula: {
-    name: "<Name of the sync formula, not show to the user>",
-    description: "<Help text for the sync formula, not show to the user>",
+    name: "Sync$1",
+    description: "Syncs the data.",
     parameters: [
-      coda.makeParameter({
-        type: coda.ParameterType.String,
-        name: "<User-visible name of parameter>",
-        description: "<Help text for the parameter>",
-      }),
-      // Add more parameters here and in the array below.
+      // TODO: Add parameters.
     ],
-    execute: async function ([param], context) {
-      let url = "<URL to pull data from>";
-      let response = await context.fetcher.fetch({
-        method: "GET",
-        url: url,
-      });
-      let items = response.body.items;
-      // Adjust the items to fit the schema if required.
+    execute: async function (args, context) {
+      let [
+        // TODO: Unpack the parameter values.
+      ] = args;
```

Comment:
Nit, very optional, aesthetically this layout isn't my favorite, I guess maybe it feels bulky and doesn't match the one-liner that real code would have when there aren't huge number of params.

I wonder if something like is easier to look at and work with.

```typescript
// TODO: Unpack the parameter values like `like [param1, param2] = args;`
let [] = args;
```

Reply:
I've found myself gravitating to the multi-line approach because:
- It has a nice vertical symmetry with the parameters array above.
- Makes it easy to scan which parameters have default values.
That said, it does look weird when there is only a single parameter, so I've changed it to use the single line approach you outlined.